fastestimator/network.py

```diff
@@ -117,6 +117,12 @@ def _forward_batch(batch: MutableMapping[str, Any], state: Dict[str, Any], ops:
             if op.outputs:
                 write_outputs_by_op(op, batch, data)
 
+    def run_step(self, batch: Dict[str, Any]):
+        raise NotImplementedError
+
+    def transform(self, data: Dict[str, Any], mode: str, epoch: int = 0):
```

Comment:
return type

Reply:
in baseclass we don't need return type because there's no return. but I added the return type in other classes.

Reply:
it should be in base so that people know what it's supposed to return when they implement it

Reply:
also for the sake of IDE which will be looking at the base signature when hinting

Reply:
fixed